### DISCUSSION TITLE
Why is this library so inactive at the moment?

### DISCUSSION THREAD

**Discussion by mccoderpy** [mccoderpy/discord.py-message-components#17] [General]
Hey there,
You may have notice that this library looks very inactive at the moment.The last [PyPi](https://pypi.org) version is full of bugs and the support server was full of scam-messages.

It's been so inactive because I've spent the last 3 months in a clinic where I've been due to extreme PC usage (12 hours non-stop at times).
Since I'm the only one working on the library, there was nothing going on during that period. 

I did a lot of work on the library before that: implemented new stuff, fixed bugs, added comments and doc strings, etc... .
I pushed many of the bug fixes directly into the development branch. A large part of the newly implemented things is only on my PC or in the code but not documented. I've wanted to release an update for a long time, in particular to include features such as slash commands and co. also available on PyPi, but I would like to have the docs ready beforehand. In addition, I'm still in the process of switching from manually written docs to code-generated ones, which somehow didn't work out that way.

First of all: I'm out of the clinic again, but I don't have my PC back at the moment. But even if I have him back, things will only progress slowly at first, because I have to prepare and study for the final exams for my secondary school leaving certificate.

After that I try to take more care of the library.

All the best and thanks for everything 

**Mathieu**; owner and developer of discord.py-message-components

**Comment by malikmajai** [comment]
I like it more than discord.py 2.0

**Reply by mccoderpy** [reply]
Thank you very much 😁😅

**Reply by zeffo** [reply]
I like you more than discord.py 🥺

**Comment by mccoderpy** [comment]
# Update
I have now finished the exams and am just waiting for the results. 😀
